<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vooki Image Viewer</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="ImageArea" name="imageAreaWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>&amp;Zoom</string>
     </property>
     <addaction name="actionZoomIn"/>
     <addaction name="actionZoomOut"/>
     <addaction name="actionFitToWindow"/>
     <addaction name="actionOriginalSize"/>
    </widget>
    <widget class="QMenu" name="menuRotate">
     <property name="title">
      <string>Rotate</string>
     </property>
     <addaction name="actionRotateLeft"/>
     <addaction name="actionRotateRight"/>
    </widget>
    <widget class="QMenu" name="menuFlip">
     <property name="title">
      <string>Flip</string>
     </property>
     <addaction name="actionFlipHorizontally"/>
     <addaction name="actionFlipVertically"/>
    </widget>
    <widget class="QMenu" name="menuNavigation">
     <property name="title">
      <string>Navigation</string>
     </property>
     <addaction name="actionNextImage"/>
     <addaction name="actionPreviousImage"/>
    </widget>
    <addaction name="actionFullScreen"/>
    <addaction name="separator"/>
    <addaction name="menuRotate"/>
    <addaction name="menuFlip"/>
    <addaction name="menuZoom"/>
    <addaction name="separator"/>
    <addaction name="menuNavigation"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <widget class="QMenu" name="menuShow">
     <property name="title">
      <string>Show</string>
     </property>
     <addaction name="actionStatusBar"/>
    </widget>
    <addaction name="menuShow"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main ToolBar</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="actionOpen"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>FileSystem Navigation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>View ToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionOriginalSize"/>
   <addaction name="actionFitToWindow"/>
   <addaction name="separator"/>
   <addaction name="actionFullScreen"/>
   <addaction name="separator"/>
   <addaction name="actionPreviousImage"/>
   <addaction name="actionNextImage"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../resource/vookiimageviewer.qrc">
     <normaloff>:/images/quit</normaloff>:/images/quit</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quits the application</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="icon">
    <iconset resource="../../resource/vookiimageviewer.qrc">
     <normaloff>:/images/fullscreen</normaloff>:/images/fullscreen</iconset>
   </property>
   <property name="text">
    <string>&amp;FullScreen</string>
   </property>
   <property name="toolTip">
    <string>Shows image in FullScreen mode</string>
   </property>
   <property name="statusTip">
    <string>Shows image in FullScreen mode</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="iconText">
    <string>About...</string>
   </property>
   <property name="toolTip">
    <string>About...</string>
   </property>
   <property name="statusTip">
    <string>About the VooView</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../../resource/vookiimageviewer.qrc">
     <normaloff>:/images/zoomin</normaloff>:/images/zoomin</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="toolTip">
    <string>Zooms the displayed image in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../../resource/vookiimageviewer.qrc">
     <normaloff>:/images/zoomout</normaloff>:/images/zoomout</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="toolTip">
    <string>Zooms the displayed image out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Opens image</string>
   </property>
   <property name="statusTip">
    <string>Opens image</string>
   </property>
   <property name="shortcut">
    <string extracomment="Opens image">Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionFitToWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resource/vookiimageviewer.qrc">
     <normaloff>:/images/originalsize</normaloff>:/images/originalsize</iconset>
   </property>
   <property name="text">
    <string>Fi&amp;t to Window</string>
   </property>
  </action>
  <action name="actionStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>StatusBar</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionOriginalSize">
   <property name="text">
    <string>Original Size</string>
   </property>
  </action>
  <action name="actionRotateLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionRotateRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionFlipHorizontally">
   <property name="text">
    <string>Horizontally</string>
   </property>
  </action>
  <action name="actionFlipVertically">
   <property name="text">
    <string>Vertically</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionPreviousImage">
   <property name="icon">
    <iconset resource="../../resource/vookiimageviewer.qrc">
     <normaloff>:/images/previous</normaloff>:/images/previous</iconset>
   </property>
   <property name="text">
    <string>Previous Image</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionNextImage">
   <property name="icon">
    <iconset resource="../../resource/vookiimageviewer.qrc">
     <normaloff>:/images/next</normaloff>:/images/next</iconset>
   </property>
   <property name="text">
    <string>Next Image</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageArea</class>
   <extends>QWidget</extends>
   <header>../../src/ui/ImageArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource/vookiimageviewer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionFullScreen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFullScreenTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onQuitTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomIn</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onZoomInTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomOut</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onZoomOutTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFitToWindow</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onFitToWindowToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTreeViewDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStatusBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onStatusBarToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOriginalSize</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onOriginalSizeTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPreviousImage</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onPreviousImageTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNextImage</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onNextImageTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onQuitTriggered()</slot>
  <slot>onZoomInTriggered()</slot>
  <slot>onZoomOutTriggered()</slot>
  <slot>onFitToWindowToggled(bool)</slot>
  <slot>onFullScreenTriggered()</slot>
  <slot>onTreeViewDoubleClicked(QModelIndex)</slot>
  <slot>onStatusBarToggled(bool)</slot>
  <slot>onOriginalSizeTriggered()</slot>
  <slot>onPreviousImageTriggered()</slot>
  <slot>onNextImageTriggered()</slot>
 </slots>
</ui>
